go/arrow/array/binary.go

```diff
@@ -105,7 +114,7 @@ func (a *Binary) String() string {
 		case a.IsNull(i):
 			o.WriteString("(null)")
 		default:
-			fmt.Fprintf(o, "%q", a.ValueString(i))
+			fmt.Fprintf(o, "%q", a.ValueStr(i))
```

Comment:
this entire switch can probably be replaced with just calling `a.ValueStr(i)` right? since it already does the `a.IsNull(i)` check

Reply:
I just kept it the same. as it was. also returned it to be `ValueString` as it doesn't need any change